flutter/Graminsta/lib/personal_info.dart

```diff
@@ -0,0 +1,233 @@
+import 'package:Graminsta/models/user.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:Graminsta/core/user_service.dart';
+import 'package:Graminsta/post/personal_service.dart';
+import 'custom_icons_icons.dart';
+import 'package:Graminsta/spec/sizing.dart';
+
+class PersonalInfo extends StatefulWidget {
+  @override
+  _PersonalInfoState createState() => new _PersonalInfoState();
+}
+
+class _PersonalInfoState extends State<PersonalInfo> {
+  static UserInfo _userInfo;
+  static String _postCount;
+  static String _followingCount;
+  static String _fanCount;
```

Comment:
You can just keep the `Map<String, int> _count` instead. When you need to reference specific count, use `_count['post_count']`.

BTW, it would be better to have a `const postCountKey = 'post_count'` defined in some `constants.dart`.